genesis.json

```diff
@@ -0,0 +1,111 @@
+{
+  "genesis_time": "2019-04-25T00:00:00.000000Z",

```

Comment:
why is the genesis file hardcoded??

Reply:
I think this can just be removed, as it is not used.

fixed: https://github.com/musnit/peggy/commit/586e80686df398a42e305fc14b58f0ccacb81697